bindings/rust/wasmedge-sys/src/vm.rs

```diff
@@ -360,6 +363,25 @@ impl Vm {
         self.run_function_async(func_name, params)
     }
 
+    pub async fn run_wasm_from_file_async2(
+        &self,
+        path: impl AsRef<Path>,
+        func_name: impl AsRef<str> + Send,
+        params: impl IntoIterator<Item = WasmValue> + Send,
+    ) -> WasmEdgeResult<Vec<WasmValue>> {
+        // load
+        self.load_wasm_from_file(path)?;
+
+        // validate
+        self.validate()?;
+
+        // instantiate
+        self.instantiate()?;
+
+        // invoke
+        self.run_function_async2(func_name, params).await
+    }
+
```

Comment:
Please replace the original `run_wasm_from_file_async` with `run_wasm_from_file_async2`, but keep using the original API name `run_wasm_from_file_async`. Thanks!